content/app/github/wrappers/line-element-wrapper.js

```diff
@@ -24,10 +61,14 @@ define("content/app/github/wrappers/line-element-wrapper", function(require, exp
 				var lines = [];
 				for (var i = 0; i < domElements.length; i++) {
 					var domElement = domElements[i];
-					// TODO: Consider additional filtering.
-					lines.push(new LineElementWrapper(domElement));
+					var className = domElement.className;
+					if (className === LineElementWrapper.TAG_LINE_CLASS || className === LineElementWrapper.TAG_EXPANDABLE_BUTTON_CLASS) {
+						lines.push(new LineElementWrapper(domElement));
+					}
 				}
 
+				this._populateMissingRanges(lines);
```

Comment:
I think populateExpandableRanges is more appropriate. You could also put code above in method _populateStandardRanges.